### ISSUE TITLE
OpenGL API version / extra files

### ISSUE THREAD

**Issue by Anuskuss** [hrydgard/ppsspp#9587]
**Settings** -> **Tools** -> **System Information** -> *Version Information*

**PPSSPP 1.3**
`API Version` v4.5.13399
`Shading Language` 4.40

**PPSSPP 1.4+**
`API Version` v4.4.13399
`Shading Language` 4.40

Is this expected to go down?

##### A little side question
Since the Direct3D 11 support, **PPSSPP** comes with `d3dcompiler_47.dll` and `d3dcompiler_47_x86.dll`. From my testing, these files are not necessary and aren't shipped in the developments builds, so why are they distributed to the normal end user?

In my opinion `assets/lang/.git` (dev builds) and `assets/lang/README.md` should also be hidden by default.

**Comment by hrydgard** [comment]
Not expected to go down, that is odd. Can you reproduce easily?

Either way I'm planning to replace our OpenGL extension loader soon so things like this might change.

d3dcompiler_47.dll is only installed by default in Windows 10 and later, so it's needed. It is however installed if you have Visual Studio installed with the Windows SDK, which might be what's throwing off your testing.

**Comment by Anuskuss** [comment]
@hrydgard 
1. I could find the version which changed the API version, but you have to tell me if it's even worth the effort.
2. Yeah, that makes sense that they ain't included in the dev builds then. What about `assets/lang/.git` and `assets/lang/README.md`?

**Comment by hrydgard** [comment]
Well, not sure it's bad, but that's a bit surprising. @unknownbrackets , your change looks fine, don't understand why we'd get a lower version. Probably doesn't matter though.

That assets/lang/.git and ended up in the release builds is an oversight, I'll fix it next time. lang/README though I think is fine.

**Comment by unknownbrackets** [comment]
@hrydgard I see it.

https://github.com/hrydgard/ppsspp/pull/9451/files#diff-b264609707058210058835781994cd9bL266

Here we WERE checking the version and extensions, and then AFTERWRAD creating a new context.  So we detected version 4.5 and its extensions, but then got a context that was 4.4 and might have less extensions.  That's the bug I fixed (although I was worried about the reverse scenario.)

In that link above, we were never trying a 4.5 context and were never getting one.  We just incorrectly had metadata from one.  In practice, the extension list is probably (?) the same.

-[Unknown]
